docs/Community/Governance.md

```diff
@@ -0,0 +1,161 @@
+
+Technical Discussions
+---------------------
+
+Technical discussions generally happen in one of these places:
+
+* [on the GitHub project](https://github.com/openenclave/openenclave), either in
+  an Issue or a Pull Request
+* in the public meetings of Committees, Special Interest Groups, or Working Groups, which can be found via the [community calendar](https://lists.confidentialcomputing.io/calendar).
+* on the community mailing list or chat server, which is detailed in the [Communications](Communication.md) document
+
+While we strive for openness and transparency, it is natural that some
+discussions will also happen in private channels, or in impromptu face-to-face
+conversations (for those who work in the same building, or bump into each other
+at a conference). The outcome of such discussions should be considered
+non-binding until the information is shared through an open medium and relevant
+stake-holders have the opportunity to contribute to the discussion.
+
+All SIG and WG leaders are responsible for their groups' creation of minutes from
+any meetings which they may hold, ensuring that the meetings are open and inclusive, and
+that decisions they make are fair and transparent.
+
+If a private discussion pertains to an open issue or PR, and results in a
+decision, that item should be updated and reference given to the conversation.
+
+Project-wide technical discussions are currently handled in
+[SIG-Architecture](sig-architecture/README.md) meetings. This SIG is comprised
+of the Leads of all other SIGs and functions analogously to a "technical
+steering committee" in other projects.
+
+Issue/PR Review Process
+-----------------------
+
+The community strives to triage and respond to all incoming issues within one
+week. If you haven't heard anything by then, or an issue or PR is not getting
+any response within a reasonable time after it is triaged, please feel free to
+remind us with a ping on the thread.
+
+Security issues should be reported through a separate channel, and
+will receive a response within 24 hours. See [Reporting Security
+Issues](Contributing.md#reporting-security-issues).
+
+Committers and Contributors
+---------------------------
+
+At present, we maintain a list of [Project Committers](Committers.md) along with
+descriptions of their knowledge areas.
+
+This will be updated and replaced with a more scalable approach to
+commit-rights management that integrates with SIG/WG governance.
+
+**Proposal** The following roles would be sufficient for SIGs and WGs to self-govern.
+
+- **Chair**: the primary point of contact for a SIG or WG, responsible for the processes within the SIG; a tie-breaker in votes within the SIG, if needed
+- **Approver**: members of the SIG who are empowered to approve code changes
+- **Reviewer**: members of the SIG whose knowledge is respected by Committers, and whom are frequently asked to provide code reviews for patches
```

Comment:
Is there a difference between "Committer" as used in this line, and "Approver" as used in the previous line?  They sound synonymous, and if so can we just use one term throughout?

Reply:
```suggestion
- **Reviewer**: members of the SIG whose knowledge is respected by Approvers, and whom are frequently asked to provide code reviews for patches
```